test/parsing/test.json

```diff
@@ -3,37 +3,37 @@
     "tests": [
         {
             "description": "with equal delimiter",
-            "$ref": "./test/parsing/files/delimiter.equal.valid.txt",
-            "result": "./files/delimiter.equal.valid.json",
+            "doc": "./test/parsing/files/delimiter.equal.valid.txt",
+            "result": "./parsing/files/delimiter.equal.valid.json",
             "valid": true,
             "object": "result"
         },
         {
             "description": "with space delimiter",
-            "$ref": "./test/parsing/files/delimiter.space.valid.txt",
-            "result": "./files/delimiter.space.valid.json",
+            "doc": "./test/parsing/files/delimiter.space.valid.txt",
+            "result": "./parsing/files/delimiter.space.valid.json",
             "valid": true,
             "object": "result"
         },
         {
             "description": "full test",
-            "$ref": "./test/parsing/files/full.example.valid.txt",
-            "result": "./files/full.example.valid.json",
-            "config": "./config.json",
+            "doc": "./test/parsing/files/full.example.valid.txt",
+            "result": "./parsing/files/full.example.valid.json",
+            "config": "./parsing/config.json",
```

Comment:
I suggest creating a separate properties `data` which includes: `doc`, `result` and `config`. I'm not sure but  [@linterhub/schema](https://github.com/linterhub/schema/) repo have a JSON Schema for tests with a similar structure.

Reply:
@svoboda-rabstvo good suggestion. I changed it, please, re-check PR.